config/sample-corexy-multi-mcu-multi-steppers.cfg

```diff
@@ -0,0 +1,150 @@
+[include mainsail.cfg]
```

Comment:
please remove the gui specific include. i think it should not be in a generic example config.

Reply:
Thank you! Forgot to remove that.